tests/wpt/meta/trusted-types/block-string-assignment-to-Document-write.html.ini

```diff
@@ -1,40 +1,10 @@
 [block-string-assignment-to-Document-write.html]
-  [`document.write(string)` throws]
-    expected: FAIL
-
-  [`document.write(string, string)` throws]
-    expected: FAIL
-
-  [`document.write(string, TrustedHTML)` throws]
-    expected: FAIL
-
-  [`document.writeln(string)` throws]
-    expected: FAIL
-
-  [`document.writeln(string, string)` throws]
-    expected: FAIL
-
-  [`document.writeln(string, TrustedHTML)` throws]
-    expected: FAIL
-
-  [`document.write(null)` throws]
-    expected: FAIL
-
-  [`document.writeln(null)` throws]
-    expected: FAIL
-
-  [`document.write(string)` observes default policy]
-    expected: FAIL
-
   [`document.write(string, string)` observes default policy]
     expected: FAIL
```

Comment:
Here it expects both strings to be concatenated and then passed into the policy. However, the parser currently expects a `Vec` as well. Since the code therefore heavily relies on chunking, I don't think I should be changing that?

Reply:
Figured we can simply change the parser and make it fully spec-compliant